lib/hls/hls_parser.js

```diff
@@ -2943,6 +2966,17 @@ shaka.hls.HlsParser = class {
       endTime = partialSegmentRefs[partialSegmentRefs.length - 1].endTime;
     }
 
+    if (segmentWithByteRangeOptimization) {
+      // We can no optimize segments with GAPs or if the start byterange is
```

Comment:
I would reword this comment. It has some grammatical errors. Something like...

```js
// We cannot optimize segments with gaps, or with a start byte that is not 0.
```

Reply:
Done!